BetaCameras/Kinect4Azure/Kinect4Azure.cs

```diff
@@ -350,7 +351,7 @@ protected override void UpdateImpl()
         {
             try
             {
-                _capture = _device.GetCapture();
+                _capture = _device.GetCapture(new TimeSpan(0, 0, 0, 1, 0));
```

Comment:
Do you know `TimeSpan.FromSeconds(double)`? It's much more readable for this kind of usage.

Reply:
Haha, Hannes here, obviously.

Reply:
Yes, I know it - but since the other variant is also correct I did not request a change ;-)